### PR TITLE
new button hover changes

### PR DESCRIPTION
**Pull request by PALLAVIKHEDLE**
- Hovering on the 'New' button now shows a dropdown.
- Clicking the new button takes you to the '/editor' page.

Screenshot 2024-10-06 at 8 52 29 PM

### CODE REVIEW

File: src/components/HeaderDropdown.tsx [hunk 1]
```diff
@@ -21,17 +24,24 @@ export default function HeaderDropdown() {
...
-    <DropdownMenu>
+    <DropdownMenu open={isHovered}>
       <DropdownMenuTrigger asChild>
         <Button
           size="sm"
           variant="default"
           className="bg-blue-600 hover:bg-blue-700"
+          onMouseEnter={() => setIsHovered(true)} // Show dropdown on hover
+          onMouseLeave={() => setIsHovered(false)} // Hide dropdown when not hovering
+          onClick={() => navigate("/editor")} // Navigate on click
```
Comment (seveibar) [inline src/components/HeaderDropdown.tsx:35]: it should take you to the quickstart, not the editor

```suggestion
          onClick={() => navigate("/quickstart")} // Navigate on click
```

**Comment by seveibar** [comment]
just need to fix the formatting and the link. Run `bun format`

File: src/components/HeaderDropdown.tsx (hunk 1 above)
Comment (PALLAVIKHEDLE) [reply src/components/HeaderDropdown.tsx]: fixed

**Comment by PALLAVIKHEDLE** [comment]
Done with the changes.

**Review by seveibar — APPROVED** [review]

**Comment by seveibar** [comment]
@PALLAVIKHEDLE 🫡

**Pull request merged**